performance/full_testenv/main.go

```diff
@@ -46,7 +46,7 @@ func startBridge(connections []conf.ConnectorConfig) (*core.NATSKafkaBridge, err
 	}
 	config.NATS = conf.NATSConfig{
 		Servers:        []string{natsURL},
-		ConnectTimeout: 2000,
+		ConnectTimeout: connectTimeout,
 		ReconnectWait:  2000,
 		MaxReconnects:  5,
```

Comment:
Make infinite or configurable... Sorry I missed this earlier.

Reply:
Good catch! Added! :+1: